backend/uclapi/uclapi/settings.py

```diff
@@ -198,3 +198,9 @@
     FAIR_USE_POLICY = list(fp)
 
 REDIS_UCLAPI_HOST = os.environ.get("REDIS_UCLAPI_HOST")
+
+# Celery Settings
+CELERY_BROKER_URL = 'redis://' + REDIS_UCLAPI_HOST + ':6379/0'
```

Comment:
> all fields after the scheme are optional, and will default to localhost on port 6379, using database 0.
You can omit the other things @ChristopherHammond13, they default to 6379 and 0 for throttling anyways.